tests/test_extensions/test_ext_autodoc.py

```diff
@@ -832,9 +832,13 @@ def test_autodoc_special_members(app):
     ]
 
     # all special methods
-    options = {"members": None,
-               "undoc-members": None,
-               "special-members": None}
+    options = {
+        "members": None,
+        "undoc-members": None,
+        "special-members": None,
+    }
+    if sys.version_info >= (3, 13, 0, 'alpha', 5):
+        options["exclude-members"] = "__static_attributes__"
```

Comment:
this ok @picnixz ?

Reply:
Err yes, although I would have also accepted something like `sys.version_info[:2] >= (3, 13)` (I didn't expect that precision of version guard since I assume that tests should be against the "latest" build of the target version). So it's up to you (at least, a guard with major/minor version and sometimes major/minor/micro if there are incompatible changes between micro versions but in general major/minor is sufficient I think).

Reply:
actually github copilot autocompleted it 😆